lib/change_health/response/claim/report/report_835_service_line.rb

```diff
@@ -8,6 +8,46 @@ class Report835ServiceLine < Hashie::Trash
         property :line_item_charge_amount, required: false
         property :line_item_provider_payment_amount, required: false
         property :service_adjustments, required: false
+
+        def create_group_adjustments(service_adjustments)
+          adjustment_array = service_adjustments[:adjustments].map do |key, value|
+            {
+              adjustmentReasonCode: key,
+              adjustmentAmount: value
+            }
+          end
+          {
+            adjustmentDetails: adjustment_array,
+            adjustmentGroupCode: service_adjustments[:claim_adjustment_group_code]
+          }
+        end
+
+        def create_remark_code_adjustments(remark_codes_array)
+          adjustment_array = remark_codes_array.map do |_key, value|
+            {
+              adjustmentReasonCode: value,
+              adjustmentAmount: ""
+            }
+          end
+          {
+            adjustmentDetails: adjustment_array,
+            adjustmentGroupCode: ""
+          }
+        end
+
+        def create_adjustment_detail_array
+          adjustment_details = []
+          all_service_adjustments = self.service_adjustments
+          all_service_adjustments.each do |service_adjustments|
```

Comment:
nitpick: Could do `adjustment_details = all_service_adjustments.map` with just `create_group_adjustments(service_adjustments)` instead of pushing onto array.